src/plugins/data/server/search/opensearch_search/opensearch_search_strategy.ts

```diff
@@ -86,8 +87,11 @@ export const opensearchSearchStrategyProvider = (
           rawResponse,
           ...getTotalLoaded(rawResponse._shards),
         };
-      } catch (e) {
+      } catch (e: any) {
```

Comment:
can this be more typed?

Reply:
don't think so, but let me switch back to use `unknown`, better than `any` at least